sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/TestProcessor.cs

```diff
@@ -335,13 +333,13 @@ private void GenerateMarkdownSummary()
 
 ![skipped](https://img.shields.io/badge/status-skipped-lightgrey) **Skipped:** {SkippedTestCount}
 
-#### For more details, visit the [service dashboard]({Uri.EscapeUriString(_cloudRunMetadata.PortalUrl!)}).
+#### For more details, visit the [service dashboard]({Uri.EscapeDataString(_cloudRunMetadata.PortalUrl!)}).
```

Comment:
Does this do the same thing in all cases?  IIRC these methods had different behaviors/outputs

Reply:
I spent awhile reading through https://github.com/dotnet/runtime/issues/31387 about why this method was deprecated. I actually don't know that this string needs to be escaped at all. It appears to be properly formatted already. Although, I ran the debugger through all the test cases with a breakpoint on this and looks like there aren't any tests for this one so hard to validate for sure.

This property is used directly in other places without escaping:
- https://github.com/Azure/azure-sdk-for-net/blob/ca726a9e07ec5cb29c35539036fff7f83495721c/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/TestProcessor.cs#L84
- https://github.com/Azure/azure-sdk-for-net/blob/ca726a9e07ec5cb29c35539036fff7f83495721c/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/TestProcessor.cs#L265

This property appears to be escaped already:
- https://github.com/Azure/azure-sdk-for-net/blob/ca726a9e07ec5cb29c35539036fff7f83495721c/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Model/CloudRunMetadata.cs#L13-L16

It would be helpful if someone from this team could take a look: CC: @Sid200026 @ShreyaAnand @puagarwa

Reply:
We don't need to escape it in `GenerateMarkdownSummary` again, it was not cleaned up properly when we moved escaping into the models class.